coverage-report/README.md

```diff
@@ -0,0 +1,16 @@
+## Coverage Report
```

Comment:
Would you add the nice screenshots you prepared in this pull request description?

Reply:
Good idea, done.